zeppelin-server/src/test/java/org/apache/zeppelin/rest/AbstractTestRestApi.java

```diff
@@ -201,6 +207,10 @@ private static void start(boolean withAuth, String testClassName, boolean withKn
       LOG.info("Staring test Zeppelin up...");
       ZeppelinConfiguration conf = ZeppelinConfiguration.create();
 
+      if (withDynamicPool) {
+        System.setProperty("org.quartz.properties", "./quartz.properties");
```

Comment:
just to check, is `./` going to the right path?

Reply:
Yes, it refers to test resources